src/main/kotlin/no/nav/klage/innsyn/api/view/SakView.kt

```diff
@@ -34,6 +34,18 @@ data class SakView(
             ANKE_AVSLUTTET_I_TRYGDERETTEN, //Do we need this, when we have ANKE_KJENNELSE_MOTTATT_FRA_TRYGDERETTEN?
             ANKE_AVSLUTTET_I_KLAGEINSTANS,
         }
+
+        data class EventDocument(
+            val title: String,
+            val archiveDate: LocalDateTime,
+            val journalpostId: String?,
+            val eventDocumentType: EventDocumentType,
+        ) {
+            enum class EventDocumentType {
+                SVARBREV_KLAGE,
+                SVARBREV_ANKE,
```

Comment:
Jeg er noe usikker på dette skillet mellom svarbrev for klager og anker. Hva skal det brukes til? Kan en klage f.eks. ha noe annet enn et svarbrev som hører til klagen? Eller kan man se for seg at vi noen ganger vil returnere alle svarbrev en bruker fått i en sak? Da blir dette nyttig.